index.js

```diff
@@ -112,20 +123,17 @@ module.exports = {
 
 // Path to the template that contains the shim wrapper around the browser
 // polyfill
-var templatePath = path.resolve(__dirname + '/assets/browser-fetch.js.t');
+const templatePath = path.resolve(__dirname + '/assets/browser-fetch.js.t');
 
 
-// Returns a tree containing the browser polyfill (from `yetch` and `abortcontroller-polyfill`),
+// Returns a tree containing the browser polyfill (from `whatwg-fetch` and `abortcontroller-polyfill`),
 // wrapped in a shim that stops it from exporting a global and instead turns it into a module
 // that can be used by the Ember app.
 function treeForBrowserFetch() {
-  var fetchPath = require.resolve('yetch/polyfill');
-  var abortcontrollerPath = require.resolve('abortcontroller-polyfill');
-
-  var expandedFetchPath = expand(fetchPath, 'dist/yetch-polyfill.js');
-  var expandedAbortcontrollerPath = expand(abortcontrollerPath, 'abortcontroller-polyfill-only.js');
-
-  var polyfillTree = concat(new MergeTrees([find(expandedFetchPath), find(expandedAbortcontrollerPath)]), {
+  const fetchTree = path.dirname(require.resolve('@xg-wang/whatwg-fetch'));
```

Comment:
lets leave a comment above this line, briefly explaining why it is a fork and when we hope to get back onto the mainline.

Reply:
Maybe we can actually have an issue on the fork with this info, and have the comment here cross link that?

Reply:
Open 2 issues where the fork is being used, I will open another one under ember-fetch and cross link them.

https://github.com/pretenderjs/pretender/issues/232
https://github.com/rwjblue/ember-cli-pretender/issues/63